pkg/testsuite/testsuite.go

```diff
@@ -79,8 +79,8 @@ func NewTestSuite(testingT *testing.T, opts ...options.Option[TestSuite]) *TestS
 		optsTick:                   DurationFromEnvOrDefault(2*time.Millisecond, "CI_UNIT_TESTS_TICK"),
 		optsGenesisTimestampOffset: 0,
 		optsLivenessThreshold:      3,
-		optsMinCommittableAge:      10,
-		optsMaxCommittableAge:      20,
+		optsMinCommittableAge:      11,
+		optsMaxCommittableAge:      21,
```

Comment:
Well, the values here are just to comply with the assumptions we had in the test before we changed the definition. Now to make it compatible I just increased those values instead of adjusting the tests.
The descriptions and reasoning with examples are written in the ProtocolParameters, and after the change it's pretty simple.

Reply:
I also think it's worth the effort to actually change it. It's going to be a bit annoying but using these values might give us a headache every time we need to calculate

Reply:
Ok, I reverted it and it turns out that after merging develop, the tools that issue blocks after removing `+1` part already work correctly. I only had to increase MinCommittableAge in some tests from 1 to 2 because it didn't make sense for a block in slot 11 to commit to slot 10 :D